src/actions/LikesAction.js

```diff
@@ -0,0 +1,56 @@
+import actionType from "./actionTypes";
+import { BACKEND_DOMAIN } from "./articleActions";
+
+const token = window.localStorage.getItem("token");
+
+const LikeStatus = slug => dispatch => {
+  return fetch(`${BACKEND_DOMAIN}articles/${slug}`, {
+    method: "GET",
+    headers: {
+      "content-type": "application/json",
+      authorization: `Bearer ${token}`
+    }
+  })
+    .then(res => res.json())
+    .then(data =>
+      dispatch({
+        type: actionType.LOAD_LIKE_STATUS,
+        payload: data
+      })
+    );
+};
+
+const LikeIcon = slug => dispatch => {
```

Comment:
`LikeStatus`, `LikeIcon` and `DisLikeIcon` are not meaningful names for actions. They should indicate an action being performed for instance `getLikesStatus`.

Reply:
Okay, let me change that